solution/ui/regulations/eregs-vite/src/components/search/RegResults.vue

```diff
@@ -22,21 +22,29 @@ const partDocumentTitleLabel = (string) => string.toLowerCase();
 
 const removeQuotes = (string) => stripQuotes(string);
 const createResultLink = (
-    { headline, title, part_number, section_number, date },
+    { headline, title, part_number, section_number, date, section_title },
     base,
     query
 ) => {
     // get highlight content from headline
     const highlightedTermsArray = getTagContent(headline, "search-highlight");
-
+    const q = query.replace("%", "%25")
```

Comment:
I'd prefer something more descriptive that `q` here for a variable name

https://courses.cs.washington.edu/courses/cse154/17au/styleguide/js/naming-conventions-js.html